content/ja/blog/_posts/2024-01-15-SIG-Release-Spotlight/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: "SIGリリースのスポットライト（リリース・チーム・サブプロジェクト）"
```

Comment:
Well... I'm not sure what's the best, but maybe keeping "SIG Release" as it is would be better, because we usually call ourselves "SIG Docs" even in the Japanese context.

What do you think? cc @kubernetes/sig-docs-ja-reviews 


btw it seems there're spelling inconsistencies even if we decided leaving this

Reply:
Agree to keep the "SIG Release" as-is.
The name of "SIG Release" is used as  one word, I think. (It's also same with SIG xxxxx)

Reply:
Thank you for your comment. Keep the “SIG Release” as is.